resources/lang/id/users.php

```diff
@@ -0,0 +1,321 @@
+
+                'best' => [
+                    'title' => 'Performa Terbaik',
+                ],
+                'first' => [
+                    'title' => 'Peringkat Tempat Pertama',
+                ],
+            ],
+            'account_standing' => [
+                'title' => 'Kondisi Akun',
+                'bad_standing' => 'Akun <strong>:username</strong> tidak dalam posisi baik :(',
+                'remaining_silence' => '<strong>:username</strong> akan dapat berbicara lagi dalam :duration.',
+
+                'recent_infringements' => [
+                    'title' => 'Pelanggaran Terbaru',
+                    'date' => 'tanggal',
+                    'action' => 'aksi',
+                    'length' => 'durasi',
+                    'length_permanent' => 'Permanen',
+                    'description' => 'deskripsi',
+                    'actor' => 'oleh :username',
+
+                    'actions' => [
+                        'restriction' => 'Blokir',
+                        'silence' => 'Bungkam',
+                        'note' => 'Catatan',
+                    ],
+                ],
+            ],
+        ],
+        'info' => [
+            'discord' => 'Discord',
+            'interests' => 'Minat',
+            'lastfm' => 'Last.fm',
+            'location' => 'Lokasi Saat Ini',
+            'occupation' => 'Pekerjaan',
+            'skype' => 'Skype',
+            'twitter' => 'Twitter',
+            'website' => 'Website',
+        ],
+        'not_found' => [
+            'reason_1' => 'Pengguna mungkin telah mengubah nama penggunanya.',
+            'reason_2' => 'Akun tersebut mungkin tidak tersedia untuk sementara karena masalah keamanan atau penyalahgunaan.',
+            'reason_3' => 'Mungkin anda salah ketik!',
+            'reason_header' => 'Ada beberapa kemungkinan mengapa hal ini bisa terjadi:',
+            'title' => 'Pengguna tidak ditemukan! ;_;',
+        ],
+        'page' => [
+            'description' => '<strong>saya!</strong> adalah area pribadi yang dapat dimodifikasi di halaman profil anda.',
+            'edit_big' => 'Sunting saya!',
+            'placeholder' => 'Ketik konten halaman di sini',
+            'restriction_info' => "Anda harus menjadi <a href='".route('store.products.show', 'supporter-tag')."' target='_blank'>osu!supporter</a> untuk membuka fitur ini.",
+        ],
+        'post_count' => [
+            '_' => 'Berkontribusi :link',
+            'count' => ':count postingan forum|:count postingan forum',
+        ],
+        'rank' => [
+            'country' => 'Peringkat negara untuk :mode',
+            'global' => 'Peringkat global untuk :mode',
+        ],
+        'stats' => [
+            'hit_accuracy' => 'Akurasi Hit',
+            'level' => 'Level :level',
+            'maximum_combo' => 'Combo Maksimum',
+            'play_count' => 'Jumlah Main',
+            'play_time' => 'Jumlah Waktu Main',
```

Comment:
Total Waktu Main/Bermain
Total lebih suitable untuk waktu daripada Jumlah

Reply:
I think "Jumlah" does better job here.
Total will be more suitable if we include currency in it.

Reply:
I'm actually thinking of "Telah Bermain Selama" here :P

Reply:
as (if you choose to implement this) it'll be looking very well like this : 

Telah Bermain Selama
[...] hrs [...] mins [...] secs